File: Algorithms/Search/LinearSearch/linearSearch.js

list = []
for( let i = 0; i < 100; i++ ) list.push( Math.round( Math.random() * 100 ) )

function linearSearch( list, target )
{
    for( let i = 0; i < list.length; i++ )
    {
        if( list[i] == target ) return true
    }

    return false
}

console.log( list )
console.log( linearSearch( list , target=10) )

File: Algorithms/Search/linearSearch.py
import random
# Take Max Size Int and Mod By Max
lenOfList = 100

# Return a List of random integers between zero and current maxInt
randomIntList = [ random.randint( 0, 100 ) for _ in range( lenOfList ) ]


target = 10

def linearSearch( List, target) :
    
    if len( [ val for val in range( len(List) ) if List[ val ] == target ] ) : print( f'{target} is in list:\n {List}' )
    else : return print( f'{target} is not in list:\n {List}' )

linearSearch( randomIntList, target )